test/e2e/storage/sanity.go

```diff
@@ -312,6 +312,41 @@ var _ = deploy.DescribeForSome("sanity", func(d *deploy.Deployment) bool {
 			v.remove(vol, name)
 		})
 
+		It("could able to handle publish volume calls after a node driver restart", func() {
```

Comment:
What about:
"can publish volume after a node driver restart"
or, if _handle_ and _call_ are important:
"can handle PublishVolume calls after a node driver restart"

Reply:
> What about:
> "can publish volume after a node driver restart"

Sounds good, I made this change.